### ISSUE TITLE
value_factor not working 

### ISSUE THREAD

**Issue by mindlessbeing** [kalkih/mini-graph-card#525]
I can't seem to get value_factor to do anything. An example would be great, though I can't imagine I missed any formatting that could work..

value_factor: 3
value_factor: 1.5
value_factor: -3
value_factor: '3'
value_factor: 0.3

It always stays the same value. Tried it on several different sensors. I must be doing something wrong, but couldn't figure out what with the documentation. Any help would be appreciated! 

Basically I just want to to display my KB/s in MB/s
So, 

value_factor: 3

right?

**Comment by kalkih** [comment]
Should be corect usage, see usage example here: https://github.com/kalkih/mini-graph-card/pull/362

Are you on the latest version of the card?

**Comment by mindlessbeing** [comment]
Yes - 0.9.4

![Screenshot_20210102-142728_Home Assistant](https://user-images.githubusercontent.com/67077556/103458275-b5ca2b00-4d06-11eb-8d4e-da867419ae3e.jpg)
![Screenshot_20210102-143020_Home Assistant](https://user-images.githubusercontent.com/67077556/103458320-2cffbf00-4d07-11eb-81f5-40ab34883571.jpg)
![Screenshot_20210102-143037_Home Assistant](https://user-images.githubusercontent.com/67077556/103458322-2ffaaf80-4d07-11eb-947e-eeb9908703e6.jpg)
![Screenshot_20210102-143234_Home Assistant](https://user-images.githubusercontent.com/67077556/103458360-6cc6a680-4d07-11eb-83cb-7ba1a4323f11.jpg)

Installed through Hacs, if that matters... Also tried re-adding, didn't help.

**Comment by kalkih** [comment]
This option is not supported in the entity configuration, only in the root card configuration
Which I guess should be fine in your case since the two entities is of the same unit.

**Comment by mindlessbeing** [comment]
Wow, I should've read which section value_factor applies to... I could've saved you the time :facepalm:.

Thanks for the help, much appreciated!

**Comment by kalkih** [comment]
No worries, the doumentation is long and not that well formatted so I totally get it 🙂

**Comment by donparlor** [comment]
@kalkih Is there a way I can use value_factor to convert minutes into hours? My sensor provide me with minutes, and I want to display as hours. Is the value_factor the way to do it?
